omod/src/main/resources/apps/registrationapp_app.json

```diff
@@ -10,7 +10,7 @@
         "extensionPointId": "org.openmrs.referenceapplication.homepageLink",
         "type": "link",
         "label": "New Registration App",
-        "url": "registrationapp/registerPatient.page?appId=referenceapplication.registrationapp.registerPatient",
```

Comment:
I forgot to change the app id here... this should be the same as the id defined above on line 3, that's why your changes weren't being pushed up...